common/Server/eos/core/eos_core.sqf

```diff
@@ -60,8 +60,12 @@ if (!(getmarkercolor _mkr == "colorblack"))then {
 										if (!surfaceiswater _mPos) then {
 											0=[_mPos,units _aGroup,_mkrX,0,[0,20],true,true] call shk_fnc_fillhouse;
```

Comment:
Let's use the garrison waypoint here if lambs is loaded. Teleport and random exit reason